lua/messages/init.lua

```diff
@@ -10,9 +10,9 @@ local create_commands = function()
   local command_name = require('messages.config').settings.command_name
   vim.api.nvim_create_user_command(
     command_name,
-    function(opts) require('messages.api').capture_cmd(opts.args) end,
+    function(opts) require('messages.api').capture_cmd(opts.args or "messages") end,
```

Comment:
I'm not sure this works since args can be an empty string which is not "falsy" unless I'm mistaken. Maybe we can do this inside `capture_cmd` instead to also support it through lua?

So eg
```lua
if cmd == '' then
  cmd = 'messages'
end
```